packages/elements-react/src/context/form-state.ts

```diff
@@ -30,6 +30,9 @@ export type FormStateAction =
       type: "action_select_method"
       method: UiNodeGroupEnum
     }
+  | {
+      type: "action_method_selector"
```

Comment:
For the future: the naming of these types is confusing to me (select_method, method_active, action_select_method, action_method_selector), I usually have to check in the actual flow to see what state we're in. But it also doesn't matter too much as it's just internals in elements. It only makes working with the state a bit brittle.

Reply:
Yea I wasn't sure about this name, either.

Until we have proper states in Kratos this will need to work though, and even then we still have to manage these states in the client in some cases, to be able to implement the method chooser. 

What about this specifically is confusing? Overall this is a fairly basic reducer..

Reply:
Knowing what `type` value has which effect / meaning!

Reply:
https://github.com/ory/elements/pull/531